### PR TITLE
change to parsing format

### PR DESCRIPTION
**Pull request by mjburton11**

### CODE REVIEW

**Comment by bqpd** [comment]
There's a lot of new data in this pull too!

File: solar/solar.py [hunk 1]
```diff
@@ -304,56 +307,66 @@ def setup(self, static, state):
...
+    latitude        [deg]       earth latitude
+    day                         day of the year [Jan 1st = 1]
+
+    Variables
+    ---------
+    Vwind                   [m/s]       wind velocity
+    V                       [m/s]       true airspeed
+    rho                     [kg/m**3]   air density
+    mu          1.42e-5     [N*s/m**2]  viscosity
+    ESirr       self.esirr  [W*hr/m^2]  solar energy
```
Comment (bqpd) [inline solar/solar.py]: be consistent, use either ^ or ** for exponentiation?

**Comment by mjburton11** [comment]
@bqpd can you help me figure out why it's not finding my airfoil file?

**Comment by bqpd** [comment]
hmm!

**Comment by bqpd** [comment]
`IOError: File /Users/jenkins/workspace/gpkit_ResearchModel_solar_PR/buildnode/macys_Sierra/optimizer/mosek/dai1336a.csv does not exist`

You probably want it to be looking in `gassolar/.../dai1336a.csv`, not the root directory!

**Comment by mjburton11** [comment]
This is where I have it looking `dirname(abspath(__file__)) + sep + "dai1336a.csv"`, but can't seem to find it there.

**Comment by mjburton11** [comment]
@bqpd do you think I could fix this problem if installed this repo?

**Comment by bqpd** [comment]
you could probably reproduce this on your computer?

**Comment by mjburton11** [comment]
test this please

**Comment by mjburton11** [comment]
@bqpd any idea why this might have failed on reynolds?

**Comment by mjburton11** [comment]
test this please

**Comment by bqpd** [comment]
"Caused: java.io.IOException: Backing channel 'reynolds' is disconnected." from https://acdl.mit.edu/csi/job/gpkit_ResearchModel_solar_PR/buildnode=reynolds,optimizer=mosek/40/console

**Comment by bqpd** [comment]
looks like a network issue or something

**Comment by mjburton11** [comment]
oh, so if I just rerun it will probably work? Cause it work just a second ago and the only I updated was the README

**Comment by bqpd** [comment]
yup!

**Pull request merged**